lib/livebook_web/live/live_helpers.ex

```diff
@@ -0,0 +1,42 @@
+defmodule LivebookWeb.LiveHelpers do
```

Comment:
Maybe call it `UserHelpers`? Otherwise this can quickly become a mix of unrelated functionality... but it is also fine to wait. :)

Reply:
Good point!